### PR TITLE
feat:created a basic api docker container

### PR DESCRIPTION
**Pull request by Anirudhpk2001**

### CODE REVIEW

**Review by KaranBantia — APPROVED** [review]
LGTM

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+flask=2.1.0
```
Comment (KaranBantia) [inline requirements.txt]: should be ==

**Review by KaranBantia — CHANGES_REQUESTED** [review]

File: DOCKER [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Use the official Python image as a base
```
Comment (KaranBantia) [inline DOCKER]: change file name to Dockerfile

**Review by KaranBantia — CHANGES_REQUESTED** [review]

**Comment by Anirudhpk2001** [comment]
Fixed!!! Can you Review now.

File: requirements.txt (hunk 1 above)
Comment (Anirudhpk2001) [reply requirements.txt]: Done

**Review by KaranBantia — APPROVED** [review]

**Review by sachi097 — CHANGES_REQUESTED** [review]
Document the api - documentation must include how to access the api (parameters) and a sample response.

**Pull request merged**